CONTRIBUTING.md

```diff
@@ -216,45 +207,44 @@ Then you just add a line to every git commit message:
-unrelated ask a maintainer, some tests are flaky and pass on a re-run.
+Keep an eye out for the CI results on the PR.
+If all is well then all tasks should succeed.
+On some repos the CI tests can take several hours to finish.
+If something failed, try to take a look at the logs to see if that seems related to your change or not.
+Then try to fix your code or the test depending on what you think is right.
+If you are unsure or think it is unrelated, ask a maintainer.
+Some tests are flaky and will pass on a re-run.
 
-After the reviewer/maintainer took a look they either write a comment stating `LGTM` (looks good to me)
-and approve the PR, in which case you do not need to do any further changes, or they write a comment
-with review feedback that you should address. Note that most changes require two reviews so only the
-second reviewer will actually merge the PR.
+After the reviewers and maintainers take a look, they will either write a comment stating `LGTM` (looks good to me) and approve the PR, in which case you do not need to do any further changes, or they write a comment with review feedback that you should address.
+Note that most changes require two reviews so only the second reviewer will actually merge the PR.
 
-If changes were requested do them locally in your branch and the amend them into the commit from the PR,
-you can use `git commit -a --amend` for that. This will add the current changes to the previous commit.
-Please do not push extra commits that say things like "apply code review" or "fix x" where x is a bug
-introduced in a commit from your PR. In that case always squash the change into the right commit to keep
-the git history clean. Our projects merge the commits as is and will will not squash them on merge to
-preserve the full original context.
+If changes were requested, make them locally in your branch and the amend them into the commit from the PR.
+You can use `git commit -a --amend` for that.
+This will add the current changes to the previous commit.
+Please do not push extra commits that say things like "apply code review" or "fix x" where x is a bug introduced in a commit from your PR.
+In that case, always squash the change into the right commit to keep the git history clean.
+Our projects merge the commits as is and will will not squash them on merge to preserve the full original context.
 
 ### Rebasing
 
-When you created a branch to work on the fix/feature it no longer will be updated with the latest changes
-from the upstream `main` branch. In order to keep your branch up to date you should rebase.
+When you create a branch to work on a fix or feature, it no longer will be updated with the latest changes from the upstream `main` branch.
+In order to keep your branch up to date you should rebase.
 
-In order to do so add the upstream repo as remote in git, i.e. for containers/common use:
+In order to do so add the upstream repo as remote in git, i.e. for containers/netavark use:
```

Comment:
I think it's an unnecessary change. Maybe it could be like this:
```suggestion
In order to do so add the upstream repo as remote in git, i.e. for containers/<repository-name> use:
```